src/header/TransferBench.hpp

```diff
@@ -2088,6 +2272,1114 @@ namespace {
     return ERR_NONE;
   }
 
+// IB Verbs-related functions
+//========================================================================================
+
+#ifndef NO_IBV_EXEC
+#define MAX_SEND_WR_PER_QP 12
+#define MAX_RECV_WR_PER_QP 12
+const unsigned int rdmaFlags = IBV_ACCESS_LOCAL_WRITE    |
+                               IBV_ACCESS_REMOTE_READ    |
+                               IBV_ACCESS_REMOTE_WRITE   |
+                               IBV_ACCESS_REMOTE_ATOMIC;
+#define IB_PSN  0
+#define INIT_ONCE(flag)\
+  do {                 \
+  if(flag) return;     \
+  else flag = true;    \
+  } while(0);
+
+const  uint64_t WR_ID = 1789;
+static ibv_device** deviceList = nullptr;
+static int RdmaNicCount        = -1;
+static std::vector<std::string> IbDeviceBusIds;
+static std::vector<std::set<int>> NicToGpuMapper;
+static std::vector<int> GpuToNicMapper;
+static std::vector<std::string> DeviceNames;
+static int GpuCount;
```

Comment:
GpuCount should be replaced by just calls to TransferBench::GetNumExecutors(EXE_GPU_GFX);